modules/check_in/config/initializers/statsd.rb

```diff
@@ -46,4 +54,12 @@ def check_in_type(params)
     check_in_param = params.dig(:session, :check_in_type) if check_in_param.nil?
     check_in_param == 'preCheckIn' ? 'pre_check_in' : 'check_in'
   end
+
+  def metric_prefix(headers, params)
+    check_in_param = params[:checkInType] || params.dig(:session, :check_in_type)
+
+    prefix = check_in_param == 'preCheckIn' ? 'pre_check_in' : 'check_in'
+    prefix += '.synthetic' if headers.key?('Sec-Datadog')
```

Comment:
Pls clarify. Are we planning to continue including synthetic logs in StatsD and filter them out in datadog dashboard?

Reply:
yes